### PR TITLE
ignoring link checking for Azure links and updating OWASP Zap link

### PR DESCRIPTION
**Pull request by walteck**

### CODE REVIEW

File: practices/cloud-services.md [hunk 1]
```diff
@@ -24,7 +24,7 @@
...
+  - Favour global-scoped (e.g. [CloudFront](https://aws.amazon.com/cloudfront/) or [Front Door](https://azure.microsoft.com/en-gb/pricing/details/frontdoor/)) or region-scoped services (e.g. [S3](https://aws.amazon.com/s3/), [Lambda](https://aws.amazon.com/lambda/), <!-- markdown-link-check-disable -->[Azure Functions](https://azure.microsoft.com/en-gb/services/functions/)) to availability-zone (AZ) scoped (e.g. [VMs](https://azure.microsoft.com/en-gb/services/virtual-machines/)<!-- markdown-link-check-enable-->, [RDS DBs](https://aws.amazon.com/rds/)) or single-instance services (e.g. [EC2 instance storage](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html)).
```
Comment (stefaniuk) [inline practices/cloud-services.md:27]: These issues seem to be intermittent and are not specific to these links. Can we set the [`markdown-check-links`](https://github.com/NHSDigital/software-engineering-quality-framework/blob/main/.github/workflows/markdown-check-links.yml) workflow to continue on error and use it for feedback only?

**Review by stefaniuk — CHANGES_REQUESTED** [review]

**Comment by regularfry** [comment]
@walteck you might want [this](https://github.com/NHSDigital/software-engineering-quality-framework/pull/310), which seems to be a bit more reliable.  I'll be interested to see if it becomes intermittent over time.

Two observations at this point:

1. We are tying correctness of our own changes to state we don't control. Someone else can make our pipeline break and that's not great. But also it's a check at a single point in time, where what we really want is continuous monitoring.
2. The failures we have seen are false positives.

**Comment by walteck** [comment]
These changes have already made it into main through another route therefore I would propose we close this PR and branch. @regularfry any objections?

**Comment by regularfry** [comment]
Yes, these are fixed elsewhere.

**Pull request closed without merging**